mlrun/model_monitoring/db/tsdb/timescaledb/timescaledb_connector.py

```diff
@@ -71,6 +78,28 @@ def __init__(
 
         self.profile = profile
 
+        # Determine the target database name
+        self.database = self._determine_database_name(profile)
+
+        # Update profile to use the determined database name
+        # This ensures the connection uses the correct database
+        if profile.database != self.database:
+            logger.info(
+                "Auto-generated database name for TimescaleDB",
+                original_database=profile.database,
+                target_database=self.database,
```

Comment:
Why are we calling it "target database"? Why target?

Reply:
Fair point, renamed to just `database` for clarity.